### PR TITLE
define allowed groups for each page

### PR DESCRIPTION
**Pull request by junminahn**
define allowed groups for each page and add mock resolver for userGroups in relay testing

### CODE REVIEW

File: app/lib/user-groups.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const GROUP_META = require('../data/groups.json');
+
+const getUserGroupLandingRoute = groups => {
+  let priorityGroup = GROUP_META[groups[0]];
+
+  for (let x = 1; x < groups.length; x++) {
+    const curr = GROUP_META[groups[x]];
```
Comment (matthieu-foucault) [inline app/lib/user-groups.js]: This may be undefined as "Realm Administrator" may not be the first group in the list

Comment (junminahn) [reply app/lib/user-groups.js]: I assumed that Realm Administrator does not have access to the portal, but I will add it into group list

Comment (matthieu-foucault) [reply app/lib/user-groups.js]: I think this function needs to be able to handle and ignore unknown groups.

Comment (junminahn) [reply app/lib/user-groups.js]: definitely, it shouldn't crash. I am thinking to set unkown groups to just guest

**Pull request merged**